### ISSUE TITLE
Seems it only support one path in <symbol>?

### ISSUE THREAD

**Issue by eschao** [monoguerin/vue-svg-sprite-loader#4]
Tried to load sprite and found some icon is not integrity, checked the codes and found:
```js
var path = symbol && symbol.path && symbol.path[0] && symbol.path[0].$ && symbol.path[0].$.d;
```
Seems only one path is outputted, why?

**Comment by eschao** [comment]
@monoguerin I created a pull request to fix it: https://github.com/monoguerin/vue-svg-sprite-loader/pull/5 please take a look. Thanks

**Comment by monoguerin** [comment]
@eschao just published version 1.1.0 with the fix for this, thanks
